### PR TITLE
#235: update readme for ios setup

### PR DESCRIPTION
**Pull request by socialcode-rob1**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -103,21 +105,46 @@ $ sudo dnf -y install nodejs
...
+
+Additional Requirements
+
+- [bundler](https://bundler.io)
+
+To run Mobidex on the iOS simulator from XCode, follow these steps.
+
+1. From the Mobidex root  directory, install all the required javascript packages.
+
+  `npm install`
```
Comment (generalpiston) [inline README.md]: Maybe something like: `Make sure you've ran the setup steps above, first`.

File: README.md [hunk 2]
```diff
@@ -103,21 +105,46 @@ $ sudo dnf -y install nodejs
...
+
+  `pushd ios && bundle install --path .gems && bundle exec pod install && popd`
+
+3.  Open `ios/mobidex.xcworkspace` in XCode
+
+4.  Select **Build Settings** for `Pods > secp256k1_swift` target and do the following:
+    > Tip: You'll notice there are 2  pods with similar names, we are only working with the one that has the underscore 'secp256k1_swift'
+
+    - Find and set **Use Header Maps** to "No"
+    - Find **Header Search Paths**
```
Comment (generalpiston) [inline README.md]: One of the below should be `User Header Search Paths` I believe.

Comment (socialcode-rob1) [reply README.md]: For two options exist.  

- Header Search Paths --> is used for frameworks/libraries regardless of system framework or 3rd party
- User Header Search Paths --> is for project headers being compiled

Only, "Header Search Paths" only worked for me.  When using "User Header Search Paths" I get an error... 
`secp256k1.h' file not found with <angled>...`

File: README.md (hunk 1 above)
Comment (socialcode-rob1) [reply README.md]: Will replace step 1

**Comment by generalpiston** [comment]
Thanks @socialcode-rob1 !

**Pull request merged**